README.md

```diff
@@ -42,6 +45,28 @@ module "hugosite" {
 }
 ```
 
+## Using Custom Error Responses from CloudFront
+Cloudfront allows you to override error responses if desired. This is useful when hosting Single Page Apps on S3 
+and want to leverage the default error document to route all requests to your index, but prevent S3 from returning
+a 404 error to the browser. Here is an example of replacing 404 errors with 200 OK responses. 
+
+```hcl
+module "hugosite" {
+  source              = "github.com/fillup/terraform-hugo-s3-cloudfront"
+  aliases             = ["www.domain.com", "domain.com"]
+  bucket_name         = "www.domain.com"
+  cert_domain         = "*.domain.com"
+  deployment_user_arn = "arn:aws:iam::111122223333:person"
+  default_root_object = null
```

Comment:
Not sure the correct syntax here, but the upgrade didn't like `null`

Reply:
it's working fine in my tests this way, so I'll leave it